examples/pyplots/pyplot_three.py

```diff
@@ -1,10 +1,11 @@
 """
-============
-Pyplot Three
-============
+=======================
+Three lines with pyplot
+=======================
 
-Plot three line plots in a single call to `~matplotlib.pyplot.plot`.
+Make three line plots with a single call to `~matplotlib.pyplot.plot`.
```

Comment:
Optional (but sounds more straight forward to me):

```suggestion
Plot three lines with a single call to `~matplotlib.pyplot.plot`.
```

Reply:
they're line2ds, but they're not really "lines", and I think it's better not to get into that here.

Reply:
The "three line plots" makes it seem like there are 3 Axes (i.e., each on a separate 'plot', even if we know we don't call them 'plots'), whereas "three lines" sounds like it's all on one Axes. I don't think it has the connotation of being a `Line2D` specifically.

Reply:
You could also say „plot three datasets“ if you think „lines“ is too problematic.

Reply:
went for tim's suggestion, thanks.